builds/misc/images.yaml

```diff
@@ -156,7 +156,7 @@ jobs:
       - template: templates/image-linux.yaml
         parameters:
           name: Metrics Collector
-          imageName: azureiotedge-metrics-collector
+          imageName: azureiotedge-test-metrics-collectorr
           project: MetricsCollector
```

Comment:
I think it would be nice to rename MetricsCollector to something that indicates this is now the test-only version.

Reply:
Renamed.

Reply:
@and-rewsmith You're only talking about changing the `name` field in the template parameters, right? _Not_ going into `edge-modules/MetricsCollector/` and changing the name there (which might be a good idea, but should happen in a different PR)?

Reply:
@damonbarry yeah I am just talking about changing the name field.